www/client.js

```diff
@@ -9,13 +9,18 @@ let app = new Vue({
         hippoName: null,
         score: null,
         isPlaying: true,
+        noseGoes: {
+            isActive: false,
+            marbleX: 0,
+            marbleY: 0,
+        },
     },
 
     methods: {
         feedMe: function () {
             // If the user taps after they've lost, don't do anything.
             // TODO: Can we have Vue remove the binding when `isPlaying` is false?
-            if (!this.isPlaying) {
+            if (!this.isPlaying || this.noseGoes.isActive) {
```

Comment:
Can't eat while noseGoes is active?

I'd also like to test how continuing to be able to eat might work. Some players might neglect to look down or try to gain some extra points before dismissing the poison.